features/course/people/add_people.feature

```diff
@@ -0,0 +1,26 @@
+Feature: Course people page
```

Comment:
@ORIGIN-Ticket-009
Feature: Add People Page
As a logged instructor, on the course people page, I want to be able to add students or teachers by email. All the students and teachers should receive the proper invitation to the course.